### PR TITLE
fix(chakra-ui): use isLoading in RefreshButton

### LINKED ISSUE #5002: [BUG] Received `false` for a non-boolean attribute `loading`
### Describe the bug

When rendering `RefreshButton` from the `refine/chakra-ui` package warnings appear in the console.

### Steps To Reproduce

Render `RefreshButton` component from the `refine/chakra-ui` package.

### Expected behavior

No warnings in the console.

### Screenshot

image

### Desktop

_No response_

### Mobile

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by adjsky**
closes #5002

In addition, the missing loader on invalidation is fixed with this PR.

### Self Check before Merge

Please check all items below before review.

-   [x] Corresponding issues are created/updated or not needed
-   [x] Docs are updated/provided or not needed
-   [x] Examples are updated/provided or not needed
-   [x] TypeScript definitions are updated/provided or not needed
-   [x] Tests are updated/provided or not needed
-   [x] Changesets are provided or not needed

### CODE REVIEW

**Review by alicanerdurmaz — APPROVED** [review]

File: packages/chakra-ui/src/components/buttons/refresh/index.tsx [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+    useResource,
+    useTranslate,
 } from "@refinedev/core";
 import {
     RefineButtonClassNames,
     RefineButtonTestIds,
 } from "@refinedev/ui-types";
-import { IconButton, Button } from "@chakra-ui/react";
 import { IconRefresh } from "@tabler/icons";
+import React from "react";
```
Comment (BatuhanW) [inline packages/chakra-ui/src/components/buttons/refresh/index.tsx:14]: Could you move React import back to top? It seems these changes aren't relevant. 🙏🏼

**Review by BatuhanW — APPROVED** [review]
Thanks for the PR 🎉, added a minor comment.

File: packages/chakra-ui/src/components/buttons/refresh/index.tsx (hunk 1 above)
Comment (adjsky) [reply packages/chakra-ui/src/components/buttons/refresh/index.tsx]: These changes were made by the pre-commit hook, i just moved them to a separate commit.
Rebased the branch, only left the commit with the relevant changes.

**Comment by BatuhanW** [comment]
Hello @adjsky thanks for the contributon! As a final step, could you add patch changeset, so we can proceed?

https://refine.dev/docs/contributing/#changeset

**Comment by adjsky** [comment]
Done.

**Review by aliemir — APPROVED** [review]

**Pull request merged**